.github/ISSUE_TEMPLATE/BUG_REPORT.yml

```diff
@@ -0,0 +1,31 @@
+name: Bug Report
```

Comment:
Will it be clear that the term 'bug' denotes an issue in documentation content, since it usually denotes an issue with the runtime behavior of a program?

Reply:
(Sorry if this double posts — I replied earlier this week, but now I don't see anything.)

I think it should be clear, in part because of the "Something is missing or incorrect" line below it, it but open to wording suggestions.  GitHub already uses "issue" as the generic term, so I'd want to avoid that — otherwise, that would have been a good term here.  Is "Problem" better?

Reply:
I agree that using the term `bug` sounds like a coding error instead of a problem with the content, maybe `Content issue` or `Content improvement` is more self-descriptive

Reply:
Changed in 33e13c9e.